releases/v0.8.0.md

```diff
@@ -0,0 +1,93 @@
+# Release Notes for v0.8.0
+
+Release Date: November 9th, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+* Upstream containerd supported by all deployment types except enclave-cc.
+  * This release includes the Nydus snapshotter (for the first time) to support upstream containerd.
+  * In this release images are still pulled inside the guest.
+  * **Nydus snapshotter requires the following annotation for each pod `io.containerd.cri.runtime-handler: <runtime-class`.**
+  * Nydus snapshotter support is still evolving. See limitations section below for details.
+* A new component, the Confidential Data Hub (CDH) is now deployed inside the guest.
+  * CDH is an evolution of the Attestation Agent that supports advanced features.
+  * CDH supports sealed Kubernetes secrets which are managed by the control plane, but securely unwrapped inside the enclave.
+  * CDH supports connections to both KBS and KMS.
+* New architecture of Attestation Agent and CDH allows a client to deploy multiple KBSes.
+  * One KBS can be used for validating evidence with the Attestation Service while another can provide resources.
+* Pulling from an authenticated registry now requires `imagePullSecrets`.
+
+**Peer Pods**
+* `peerpod-ctl` tool has been expanded.
+  * Can check and clean old peerpod objects
+  * Adds SSH authentication support to libvirt provider
+  * Supports IBM cloud
+* Support for secure key release at runtime and image decryption via remote attestation on AKS
+* Added AMD SEV and IBM s390x support for the Libvirt provider
+* Container registry authentication now bootstrapped from userdata.
+* Enabled public IP usage for pod VM on AWS and PowerVS providers
+* webhook: added IBM ppc64le platform support
+* Support adding custom tags to podvm instances
+* Switched to launching CVM by default on AWS and Azure providers
+* Added rollingUpdate strategy in cloud-api-adaptor daemonset
+* Disabled secureboot by default
+
+## Hardware Support
+
+Confidential Containers is tested with attestation on the following platforms:
+
+* Intel TDX
+* AMD SEV(-ES)
+* Intel SGX
+
+The following platforms are untested or partially supported:
+
+* IBM Secure Execution (SE) on IBM zSystems (s390x) running LinuxONE
+* AMD SEV-SNP
+
+The following platforms are in development:
+
+* ARM CCA
```

Comment:
ARM CCA has been supported I think, pls refer to:

- https://github.com/confidential-containers/attestation-service/pull/90
- https://github.com/confidential-containers/guest-components/pull/321
- https://github.com/confidential-containers/kbs/pull/76

A native verifier is on the way, but this is a different story.

Reply:
thanks Tobin!